github/repos_test.go

```diff
@@ -926,3 +926,43 @@ func TestPullRequestReviewsEnforcementRequest_MarshalJSON_nilDismissalRestirctio
 		t.Errorf("PullRequestReviewsEnforcementRequest.MarshalJSON returned %+v, want %+v", string(json), want)
 	}
 }
+
+func TestRepositoriesService_ListAllTopics(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/topics", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		fmt.Fprint(w, `{"names":["go", "go-github", "github"]}`)
+	})
+
+	got, _, err := client.Repositories.ListAllTopics(context.Background(), "o", "r")
+	if err != nil {
+		t.Errorf("Repositories.ReplaceAllTopics returned error: %v", err)
+	}
+
+	want := &Topics{Names: []string{"go", "go-github", "github"}}
+	if !reflect.DeepEqual(got, want) {
+		t.Errorf("Repositories.ReplaceAllTopics returned %+v, want %+v", got, want)
```

Comment:
You can see the comment was in the right place on the [original commit where I left it](https://github.com/google/go-github/pull/672/commits/43efbcca65900900ec7cda9ccf16dcbf8d47364e).